src/checkers/inference/InferenceMain.java

```diff
@@ -440,10 +449,10 @@ public DefaultResultHandler(Logger logger) {
         }
 
         @Override
-        public void handleCompilerResult(boolean success, String javacOutStr) {
-            if (!success) {
-                logger.severe("Error return code from javac! Quitting.");
-                logger.info(javacOutStr);
+        public void handleCompilerResult(Main.Result result, String javacOutStr) {
+            if (result != Main.Result.OK) {
+                logger.severe("Non-OK return code from javac! Quitting. Result code is: " + result);
+                logger.severe(javacOutStr);
```

Comment:
`logger.info(javacOutStr)` doesn't output outpustream to log file(because log level is warning). And there is not reason that we use two logging levels for the messages due to the same reason.